python/lsst/ts/scheduler/model.py

```diff
@@ -0,0 +1,46 @@
+import time
+from SALPY_scheduler import *
```

Comment:
I don't think it is a good idea to `import *`. Here you are using only `SAL_scheduler` and `scheduler_logevent_validSettingsC()`, so its better to import them individually.

Reply:
good point, done